readme.md

```diff
@@ -278,7 +278,14 @@ to start with “kernel documentation”, which is dense).
     first edition of a workshop to get started with XDP.
 -   [XDP for the Rest of Us](https://www.netdevconf.org/2.2/session.html?gospodarek-xdp-workshop)
     from the same authors, at Netdev 2.2: second edition, with new contents.
--   [Notes on bpf](https://blogs.oracle.com/linux/notes-on-bpf-1) Some wonderful tutorial or notes about (e)bpf from oracle.
+-   Generic Documentation and Presentions from oracle's Greg Marsden
```

Comment:
What I meant with `Generic Documentation and Presentations` was not for you to add it in the description, but instead to move the links to the related section of the README.md file (instead of the current “Tutorials” section :).

Missing capital letter on “Oracle”. By the way, I wouldn't mention Oracle at all, links to other presentations do not focus on authors' employers.